wafer/settings.py

```diff
@@ -148,6 +148,8 @@
     'wafer.sponsors',
     'wafer.pages',
     'wafer.tickets',
+    # Django isn't finding the overridden templates
+    'registration',
```

Comment:
Could you elaborate a bit on the moving of `registration` to after the `wafer.*` entries?